scripts/create-tables.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+aws dynamodb create-table \
+    --table-name event_journal \
+    --attribute-definitions \
+        AttributeName=pid,AttributeType=S \
+        AttributeName=seq_nr,AttributeType=N \
+        AttributeName=entity_type_slice,AttributeType=S \
+        AttributeName=ts,AttributeType=N \
+    --key-schema \
+        AttributeName=pid,KeyType=HASH \
+        AttributeName=seq_nr,KeyType=RANGE \
+    --provisioned-throughput \
+        ReadCapacityUnits=5,WriteCapacityUnits=5 \
+    --global-secondary-indexes \
+            "[
+                {
+                    \"IndexName\": \"event_journal_slice_idx\",
+                    \"KeySchema\": [
+                        {\"AttributeName\":\"entity_type_slice\",\"KeyType\":\"HASH\"},
+                        {\"AttributeName\":\"ts\",\"KeyType\":\"RANGE\"}
+                    ],
+                    \"Projection\": {
+                        \"ProjectionType\":\"ALL\"
+                    },
+                    \"ProvisionedThroughput\": {
+                        \"ReadCapacityUnits\": 5,
+                        \"WriteCapacityUnits\": 5
+                    }
+                }
+            ]"
```

Comment:
If we don't need to do any variable substitution, then cleaner to use single quotes around the JSON?

```suggestion
    --global-secondary-indexes \
      '[
         {
           "IndexName": "event_journal_slice_idx",
           "KeySchema": [
             {"AttributeName": "entity_type_slice", "KeyType": "HASH"},
             {"AttributeName": "ts", "KeyType": "RANGE"}
           ],
           "Projection": {
             "ProjectionType": "ALL"
           },
           "ProvisionedThroughput": {
             "ReadCapacityUnits": 5,
             "WriteCapacityUnits": 5
           }
        }
      ]'
```

Reply:
nice, I copied from aws doc